src/data-table/__tests__/data-table-rtl.test.js

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {mount} from 'enzyme';
+import {columns, rows} from './data-table.scenario.js';
+import {Unstable_DataTable} from '../index.js';
+import {ThemeProvider, LightTheme} from '../../index.js';
+
+describe('rendering data-table in RTL direction', () => {
```

Comment:
Can you please remove the snapshot and this unit test and convert it to a visual regression test by creating `data-table-rtl.scenario.js`? I would prefer to have a visual check rather than the jest snapshot. Those tend to just build up noise that's not meaningful

Reply:
I'll add a scenario.